src/main/java/seedu/planitarium/parser/Parser.java

```diff
@@ -115,12 +129,53 @@ public static void isValidMoney(String amount) throws NumberFormatException {
      * Returns without exception if user index is within membership quantity bounds.
      *
      * @param userIndex Person's user index.
-     * @throws NullPointerException if amount is not a valid integer.
+     * @throws NumberFormatException if amount is not a valid integer.
      * @throws IndexOutOfBoundsException if provided index is out of bounds.
      */
     public static void isValidUserIndex(String userIndex) throws NumberFormatException, IndexOutOfBoundsException {
         int checkIndex = Integer.parseInt(userIndex);
-        if (checkIndex < MIN_USER_INDEX || checkIndex >= PersonList.getNumberOfMembers()) {
+        if (checkIndex < MIN_USER_INDEX) {
+            throw new IndexOutOfBoundsException();
+        }
+        if (checkIndex >= PersonList.getNumberOfMembers()) {
```

Comment:
I think it'll be more readable to have this as it's own function. Also allows for better SLAP